src/Microsoft.Health.Dicom.Functions/Indexing/ReindexDurableFunction.Orchestration.cs

```diff
@@ -32,6 +32,7 @@ public partial class ReindexDurableFunction
         {
             EnsureArg.IsNotNull(context, nameof(context));
```

Comment:
Should we rethink this method. This is supposed to just reindex not add. 
It can change the status to "processing". But it should be added in Dicom server